pkg/kapp/app/recorded_app.go

```diff
@@ -130,6 +130,77 @@ func (a *RecordedApp) UpdateUsedGVsAndGKs(gvs []schema.GroupVersion, gks []schem
 func (a *RecordedApp) CreateOrUpdate(labels map[string]string) error {
 	defer a.logger.DebugFunc("CreateOrUpdate").Finish()
 
+	foundMigratedApp, err := a.findAndUpdate(a.fqName(), labels)
+	if err != nil {
+		return err
+	}
+
+	if foundMigratedApp {
+		a.isMigrated = true
+		return nil
+	}
+
```

Comment:
nit: Do we need this new line?

Reply:
I added this new line because the rest of the code is for when the migrated app isn't found. But yeah I do see that there are a lot of new lines now.

Reply:
Removed the newline below the declaration. Same for `RenamePrevApp`. Let me know if this looks fine.